docs/source/examples.rst

```diff
@@ -2,7 +2,7 @@ Examples
 ========
 
 Notebooks with examples of how you can use MRpro.
-All of the notebooks can directly be run via binder or colab from the repo.
+All the notebooks can be launched in Colab using the badge.
```

Comment:
Maybe we can add the global colab link
```rst
.. |colab-badge| image:: https://colab.research.google.com/assets/colab-badge.svg
    :target: https://colab.research.google.com/github/PTB-MR/mrpro
```
here is as well

Reply:
yes, please

Reply:
```suggestion
Each notebook can be launched in `Colab <https://colab.research.google.com/github/PTB-MR/mrpro)>`_
```